features/src/main/feature/camel-features.xml

```diff
@@ -490,92 +505,61 @@
     <feature name='camel-azure-cosmosdb' version='${project.version}' start-level='50'>
         <feature version='${camel.osgi.version.range}'>camel-cloudevents</feature>
         <feature version='[2.16,2.17)'>jackson</feature>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-core/1.46.0</bundle>
+        <feature>azure</feature>
         <bundle dependency='true'>wrap:mvn:com.azure/azure-cosmos/4.56.0</bundle>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-identity/1.11.2</bundle>
-        <bundle dependency='true'>mvn:org.reactivestreams/reactive-streams/${reactive-streams-version}</bundle>
-        <bundle dependency='true'>mvn:io.projectreactor/reactor-core/3.4.34</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-azure-cosmosdb/${project.version}</bundle>
     </feature>
     <feature name='camel-azure-eventhubs' version='${project.version}' start-level='50'>
         <feature version='${camel.osgi.version.range}'>camel-cloudevents</feature>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-core-amqp/2.9.1</bundle>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-core/1.46.0</bundle>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-identity/1.11.2</bundle>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-messaging-eventhubs/5.18.1</bundle>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-messaging-eventhubs-checkpointstore-blob/1.19.1</bundle>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-storage-blob/12.25.2</bundle>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-storage-common/12.24.2</bundle>
-        <bundle dependency='true'>mvn:io.projectreactor/reactor-core/3.4.34</bundle>
-        <bundle dependency='true'>mvn:org.reactivestreams/reactive-streams/${reactive-streams-version}</bundle>
+        <feature>azure-eventhubs</feature>
+        <bundle dependency='true'>wrap:mvn:com.azure/azure-storage-blob/12.25.1</bundle>
```

Comment:
There are 2 usages of `wrap:mvn:com.azure/azure-storage-blob/12.25.1` bundle, 2 as a single bundle, one along with `<bundle dependency='true'>wrap:mvn:com.azure/azure-storage-blob-changefeed/${azure-storage-blob-changefeed-version}</bundle>`
Can extract it to a separate feature as well - not sure if it is feasible